tests/spread/integration/adduser/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Integration tests for adduser 
+
+execute: |
+  rootfs="$(install-slices base-files_base bash_bins adduser_bins)"
+  chroot "$rootfs"
```

Comment:
I don't think this is working like you expect, this will chroot and exit immediately, the rest of this test runs on the host, not inside the chroot. 

You either need to run commands one-by-one prefixed by chroot or put your commands into a script, and then copy that script into the chroot and execute it